actors/runtime/indices/indices.go

```diff
@@ -237,12 +237,15 @@ func (inds *IndicesImpl) StoragePower_PledgeSlashForSurprisePoStFailure(
 	panic("")
 }
 
+// FIL deposit per sector precommit in Interactive PoRep
+// refunded after ProveCommit but burned if PreCommit expires
 func (inds *IndicesImpl) StorageMining_PreCommitDeposit(
 	sectorSize abi.SectorSize,
 	expirationEpoch abi.ChainEpoch,
 ) abi.TokenAmount {
 	PARAM_FINISH()
-	panic("")
+	const PRECOMMIT_DEPOSIT_PER_BYTE = abi.TokenAmount(0) // placeholder
```

Comment:
These placeholder comments concern me, I think they'll be very easy to overlook. We need to have no placeholders in a couple of weeks (even if the values might change for mainnet).

Reply:
agree fyi @jzimmerman